### PR TITLE
All the settings

### PR DESCRIPTION
**Pull request by Jetski5822**
At the moment only a small subset of settings get passed in to Orchard.... now all go in to the Shell Settings.

### CODE REVIEW

File: src/OrchardCore/Orchard.Environment.Shell.Abstractions/ShellSettings.cs [hunk 1]
```diff
@@ -11,7 +11,6 @@ namespace Orchard.Environment.Shell
     /// </summary>
     public class ShellSettings
     {
-        private TenantState _tenantState = TenantState.Invalid;
```
Comment (sebastienros) [inline src/OrchardCore/Orchard.Environment.Shell.Abstractions/ShellSettings.cs]: Maybe it should be kept and parsed in the ctor instead of the getter. This way the setter and the getter manipulate the Enum.

Comment (sebastienros) [inline src/OrchardCore/Orchard.Environment.Shell.Abstractions/ShellSettings.cs]: I see this was done this way before, so why did you change it?

Comment (Jetski5822) [inline src/OrchardCore/Orchard.Environment.Shell.Abstractions/ShellSettings.cs]: um... I did? I change it in this PR.... was it in the ctor before?

Comment (Jetski5822) [inline src/OrchardCore/Orchard.Environment.Shell.Abstractions/ShellSettings.cs]: I dont think we need it.... If we cant parse the state its always invalid.

Comment (sebastienros) [inline src/OrchardCore/Orchard.Environment.Shell.Abstractions/ShellSettings.cs]: There is a setter, and if it's always invalid, why parse it again? You are wrong, stop fighting.
